### ISSUE TITLE
Cannot dualboot macOS and Silverblue

### ISSUE THREAD

**Issue by osalbahr** [fedora-silverblue/issue-tracker#611]
**Describe the bug**

Cannot dualboot macOS and Silverblue on an Intel-based MacBook Pro (2017). I get the following error:

```
The following error occurred while installing the payload. This is a fatal error and installation will be aborted.

 The command 'ostree admin --sysroot=/mnt/sysimage deploy --os=fedora fedora:fedora/41/x86_64/silverblue' exited with the code 1.
```

**To Reproduce**
Please describe the steps needed to reproduce the bug:
1. Follow [QA:Testcase dualboot with macOS](https://fedoraproject.org/wiki/QA:Testcase_dualboot_with_macOS)
2. The error appears after selecting the remaining storage and attempting to install via the GUI

**Expected behavior**

Successful install.

**Screenshots**

![image](https://github.com/user-attachments/assets/bbaf05b0-566a-4761-a370-95ab9e186c52)

**OS version:**

Fedora Silverblue 41 ISO (using Fedora Media Writer).

**Additional context**

I am able to dualboot macOS and Workstation.

I have not attempted to install Silverblue on the full disk, but I can try doing that if it might help narrow down the bug. The 2017 mbp is just sitting on the side for experimentation. I plan to retire the laptop to Fedora once it cannot receive macOS security updates. Would be cool to retire it to Silverblue.

**Comment by osalbahr** [comment]
Update: I just tried installing on full disk and it worked. So it has to be something to do with dualboot. For reference, my device is `MacBookPro14,1` or "MacBook Pro (13-inch, 2017, Two Thunderbolt 3 ports)".

![image](https://github.com/user-attachments/assets/c1ef024b-b43a-446c-9941-b5ba23842db7)

**Comment by travier** [comment]
Make sure to use a distinct, empty /boot & /efi partitions for the Silverblue installation.

**Comment by osalbahr** [comment]
> Make sure to use a distinct, empty /boot & /efi partitions for the Silverblue installation.

@travier how can I do that? The way I currently install is I let the installer "claim the remaining space". I assume the installer should automatically create the necessary partitions.

I am willing to reinstall macOS and re-attempt dualbooting if it might help debugging this issue. Let me know what I should try.

**Comment by osalbahr** [comment]
The issue seems to have been resolved in the [Fedora 42 Rawhide 20241109.n.0 nightly compose nominated for testing](https://lists.fedoraproject.org/archives/list/test-announce@lists.fedoraproject.org/thread/AD2LFJQYGPEFHCBM7XR5CR26TK6SVZRT) as I am able to successfully boot into the Silverblue partition.
